### ISSUE TITLE
Stat ID Map Not Populated in League Resource

### ISSUE THREAD

**Issue by JustinMarinelli** [spilchen/yahoo_fantasy_api#55]
It seems like it would make sense for a `League` to have its `stats_id_map` property populated upon initialization. That is, after a call to `yahoo_fantasy_api.League(oauth, '{{league_id}}')`. This is not currently the case, and instead, a call to `League.stats_id_map` returns `None` immediately after initialization. Due to this issue, it's difficult to display stats on a UI consuming data returned from a call to `League.matchups()` in a human-friendly format.

A workaround I've found is to make a call to the `League`'s `_cache_stats_id_map()` method before trying to access `League.stats_id_map`. However, due to `_cache_stats_id_map` being prefixed with a "_", it seems to suggest that this method should be used ONLY internally by the `yahoo_fantasy_api` library.

The ideal behavior to me would be that this map is populated upon initialization of a `League`.

**Comment by DMcP89** [comment]
@spilchen I would agree here. We should be populating everything possible with the initial call. I can get a PR going for this if you like.

**Comment by spilchen** [comment]
Yeah, I agree. This makes sense to do.
